DigitalLearningSolutions.Web.Tests/Controllers/SuperAdmin/FaqControllerTests.cs

```diff
@@ -0,0 +1,93 @@
+namespace DigitalLearningSolutions.Web.Tests.Controllers.SuperAdmin
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Controllers.SuperAdmin;
+    using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class FaqControllerTests
+    {
+        private readonly IEnumerable<Faq> faqs = new List<Faq>
+        {
+            FaqTestHelper.GetDefaultFaq(1, weighting: 40),
```

Comment:
Here I would be tempted to define these in the setup and use soemthing more like `Builder<Faq>.CreateListOfSize(10).Build();` rather than the `FaqTestHelper`.